pkg/api/aim/runs.go

```diff
@@ -977,7 +1030,10 @@ func SearchAlignedMetrics(c *fiber.Ctx) error {
 			}
 
 			for rows.Next() {
-				var metric database.Metric
+				var metric struct {
+					database.Metric
```

Comment:
doesn't database.Metric already include Context into this struct?

Reply:
yes, but it won't be populated unfortunately, since it is a nested object and `query` is complex so you can't norally use like `Joins("Context")`.